dbus-data/org.fedoraproject.Anaconda.Addons.HelloWorld.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+
+# Start the org.fedoraproject.Anaconda.Addons.HelloWorld service.
+# Runs org_fedora_hello_world/dbus_service/__main__.py
+
+export PYTHONPATH=/usr/share/anaconda/addons
+python3 -m org_fedora_hello_world.dbus_service
```

Comment:
We might need to address Platform Python for RHEL here in the future.

Reply:
I changed this to detect its presence - see the last commit.